src/app/microfrontend/lookup.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { Microfrontend } from './microfrontend';
+
+@Injectable({ providedIn: 'root' })
+export class LookupService {
+  lookup(): Promise<Microfrontend[]> {
+    return fetch('./assets/json/mfe-routes.json').then(res => res.json())
```

Comment:
assets/routes/default-routes.json

Later this can be used to create sidebar menu items and MFE routes files can be merged with order:n parent:root/specific menu item values